src/components/Match/Overview/Timeline.jsx

```diff
@@ -164,7 +165,7 @@ export default ({ match }) => {
                         />
                       }
                       <ReactTooltip
-                        id={`event_${i}`}
+                        id={obj.type === 'firstblood' && !obj.key && !obj.player_slot ? 'notthistime' : `event_${i}`}
```

Comment:
hehe, way to hide tooltip if first blood is not in objectives and and we can't get killer and target